doc/integrator/install_application.rst

```diff
@@ -355,3 +355,31 @@ Your application should be available at:
 
 Where the ``<hostname>`` is directly linked to the virtual host,
 and the ``<instanceid>`` is the value you provided before.
+
+.. _integrator_install_application_migrate_server:
```

Comment:
Why this?

Reply:
I guess I thought I was going to create a reference to this section in another place. But finally, there isn't any reference to this, and I think we don't need one. Shall I remove it?

Reply:
I think that will be cleaner :-)

Reply:
Done in https://github.com/camptocamp/c2cgeoportal/pull/3536